moveit_ros/moveit_servo/include/moveit_servo/servo_calcs.h

```diff
@@ -38,6 +38,9 @@
 
 #pragma once
 
+// C++
+#include <mutex>
```

Comment:
I don't think you needed to add this #include because it compiled previously, and you didn't add any new mutexes